packages/treasury/test/swingsetTests/bootstrap.js

```diff
@@ -59,7 +67,9 @@ function makeBootstrap(argv, cb, vatPowers) {
     const vatAdminSvc = await E(vats.vatAdmin).createVatAdminService(
       devices.vatAdmin,
     );
-    const zoe = E(vats.zoe).buildZoe(vatAdminSvc);
+    const { zoeService: zoe, feeMintAccess } = await E(vats.zoe).buildZoe(
```

Comment:
What? Surprising. From your earlier changes, I thought `.buildZoe` still returned a zoe, as in the case I suggested renaming `makeZoe` above. Prior to this PR I'm surprised that a `buildZoe` that returns a zoe it built isn't called `makeZoe`. With this PR I am even more confused about what return value I should expect from which name. For the return value here, I'd expect `makeZoeKit`.

Reply:
This is for a test of the treasury contract, which uniquely needs the `feeMintAccess`. In the other tests, `feeMintAccess` is dropped in the destructuring, but here it is passed on.